actors/builtin/miner/monies.go

```diff
@@ -3,15 +3,49 @@ package miner
 import (
 	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
 )
 
+const InitialPledgeFactor = int64(20)
+const ExpectedFaultFactorNum = int64(214)
+const ExpectedFaultFactorDenom = int64(1000)
+const UnexpectedFaultFactor = int64(5)
```

Comment:
Please document these. 

For consistency with naming elsewhere:
- Expected -> Declared
- Unexpected -> Undeclared

Also, let's make the big.Ints for these all up front rather than re-allocating at every use.